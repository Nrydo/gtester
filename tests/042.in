3783025
632967339
106560612