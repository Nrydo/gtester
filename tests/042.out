4063107789613004